### PR TITLE
Fix #444 - Don't attempt to publish nightly snapshots from forks

### LINKED ISSUE #444: Don't attempt to publish nightly snapshots from forks
**Describe the bug**
Publication of nightly snapshots is attempted on forks.

**Stack Trace**

Workflow log: https://github.com/Gamebuster19901/litiengine/runs/4750853093?check_suite_focus=true

**To Reproduce**
1. Fork this repository
2. Wait 24 hours

**Expected behavior**
Nightly snapshots are not published from forks.

**Screenshots**
Webhook spam:
image

**Your System:**
 N/A

### PR DESCRIPTION
**Pull request by Gamebuster19901**
Also now allows the nightly snapshot to be built manually.

image

### CODE REVIEW

File: .github/workflows/snapshot.yml [hunk 1]
```diff
@@ -13,7 +14,7 @@ on:
 
 jobs:
   gradle:
-    if: github.repository_owner = 'gurklenlabs'
+    if: github.repository_owner == 'gurklenlabs'
```
[the remark this answers is not in the document]
Comment (Gamebuster19901) [reply .github/workflows/snapshot.yml]: lmao xD. Should be fixed.

**Review by nightm4re94 — APPROVED** [review]
Good to go, thank you!

**Comment by Gamebuster19901** [comment]
@weisJ sorry for the ping again, but it seems there is an issue with nightly builds, and it seems it wasn't caught before because they weren't being ran before. https://github.com/gurkenlabs/litiengine/runs/4778846110?check_suite_focus=true

**Comment by weisJ** [comment]
This is a rather bizarre error message. I'm not sure whay the `PGP` task isn't even attempted to be executed (the task doesn't even exist and isn't part of the gradle invokation). Perhaps there is a stray newline character somewhere in the repository secrets?

**Pull request merged**